src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -108,9 +122,21 @@ def _get_context(
         except TypeError:
             sdk_url = ""  # It fails if it cannot inject the version in the string
 
+        config: SdkConfig = {"dsn": key.dsn_public}
+
+        if has_debug:
+            config["debug"] = True
+
+        if has_performance:
+            config["tracesSampleRate"] = 0.1
```

Comment:
```suggestion
            config["tracesSampleRate"] = 1
```
Let's change this to `1`